uportal-war/src/main/java/org/apereo/portal/layout/IUserLayoutManager.java

```diff
@@ -98,6 +98,21 @@ public IUserLayoutNodeDescription addNode(
             throws PortalException;
 
     /**
+     * Add a node under a parent without specifying in what order it ought to be placed among
+     * siblings. Use addNode(node, parentId, nextSiblingId) to express exactly where among
+     * current children of the parent node the new node ought to be placed. Use this method to
+     * leave sibling order to the discretion of the underlying layout manager.
+     *
+     * @param nodeToAdd non-null node to add to the layout
+     * @param parentId non-null ID of parent under which to add the node
+     * @return new description of the added node, now with a new or updated ID,
+     * or null if not added.
+     * @since 5.0.0
+     */
+    IUserLayoutNodeDescription addNodeInAnyOrder(
```

Comment:
Suggest name change -- I think simply a new overload of `addNode` is a better here...

  - The "any order" part is misleading;  to me it implies rendomness
  - It's weird that this method has a longer/more complicated name than the version that accepts 3 arguments